libkmod/libkmod-elf.c

```diff
@@ -54,10 +54,14 @@ struct kmod_elf {
 	} header;
 };
 
+//#undef ENABLE_ELFDBG
 //#define ENABLE_ELFDBG 1
```

Comment:
One of both is enough as a hint ;)

Reply:
Not sure if my edit went through before your answer. Why do we need these lines at all? We have configure options now.